dub.sdl

```diff
@@ -8,13 +8,12 @@ targetPath "out/"
 dependency "inmath" version="~>1.0.2"
 dependency "vmc-d" version="~>1.1.2"
 dependency "fghj" version="~>1.0.0"
-dependency "vibe-d:http" version="~>0.9.4"
 
 configuration "default" {
     targetType "library"
 }
 
 configuration "jml" {
```

Comment:
maybe rename jml to something more descriptive.

maybe web-adaptors

Reply:
I was thinking that it made sense to change it to something like "vibe-d" or "http-adaptors" or something, i'll go with web-adaptors :)

Reply:
"jml" is the short tag for "JinsMemeLogger"
I assume that when someone decided to turn on jml adaptor, this configuration is specified from outside.